2016/05/26 06:31:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "wp-login.php".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('wp-login.php')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/wp-login.php
HTTP_REFERER=http://gettimi.com/wp-login.php
---
2016/05/26 06:31:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "administrator".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('administrator')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/administrator/
HTTP_REFERER=http://gettimi.com/administrator/
---
2016/05/26 06:36:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://gettimi.com/site/downloadAndroid
---
2016/05/26 07:01:03 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:03 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=Burger%2CItalian%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898148&_=1464245898155
---
2016/05/26 07:01:04 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=Burger%2CItalian%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898152&_=1464245898156
---
2016/05/26 07:01:11 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=Burger%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898150&_=1464245898157
---
2016/05/26 07:01:17 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:17 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=Burger%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898144&_=1464245898158
---
2016/05/26 07:01:18 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=Burger%2CDim%20Sum%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898142&_=1464245898159
---
2016/05/26 07:01:19 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:19 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=Burger%2CDim%20Sum%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898146&_=1464245898160
---
2016/05/26 07:01:19 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:19 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=Burger%2CDim%20Sum%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898138&_=1464245898161
---
2016/05/26 07:01:29 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=Burger%2CDim%20Sum%2CFrench%2CMexican%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898140&_=1464245898162
---
2016/05/26 07:01:30 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=Burger%2CDim%20Sum%2CFrench%2CMexican%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898129&_=1464245898163
---
2016/05/26 07:01:30 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=Burger%2CDim%20Sum%2CFrench%2CMexican%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898164&_=1464245898165
---
2016/05/26 07:01:32 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:32 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=Burger%2CChinese%2CDim%20Sum%2CFrench%2CMexican%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898166&_=1464245898167
---
2016/05/26 07:01:32 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:32 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=Burger%2CChinese%2CDim%20Sum%2CFrench%2CMexican%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898168&_=1464245898169
---
2016/05/26 07:01:34 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:34 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=American%2CBurger%2CChinese%2CDim%20Sum%2CFrench%2CMexican%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898170&_=1464245898171
---
2016/05/26 07:01:34 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:34 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=American%2CBurger%2CChinese%2CDim%20Sum%2CFrench%2CMexican%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898172&_=1464245898173
---
2016/05/26 07:01:36 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:36 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=American%2CBurger%2CChinese%2CDim%20Sum%2CFrench%2CMexican%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898174&_=1464245898175
---
2016/05/26 07:01:37 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:37 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=American%2CBurger%2CChinese%2CDim%20Sum%2CFrench%2CMexican%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898176&_=1464245898177
---
2016/05/26 07:01:38 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:38 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=American%2CBurger%2CChinese%2CDim%20Sum%2CFrench%2CMexican%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898178&_=1464245898179
---
2016/05/26 07:01:38 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:01:38 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 j...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=American%2CBurger%2CChinese%2CDim%20Sum%2CFrench%2CMexican%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20just%20looking%20for%20friends%20&current=Security%20&callback=jQuery111106060242655221373_1464245898180&_=1464245898181
---
2016/05/26 07:02:20 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 f...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:02:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 f...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=French%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20friends%3F%20&current=Security%2Fcollege%20&callback=jQuery111105774273385759443_1464246102706&_=1464246102719
---
2016/05/26 07:02:20 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 f...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:02:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 f...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=French%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20friends%3F%20&current=Security%2Fcollege%20&callback=jQuery111105774273385759443_1464246102714&_=1464246102720
---
2016/05/26 07:02:21 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 f...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:02:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 f...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=French%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20friends%3F%20&current=Security%2Fcollege%20&callback=jQuery111105774273385759443_1464246102712&_=1464246102721
---
2016/05/26 07:02:23 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 f...' for column 'whatsup' at row 1. The SQL statement executed was: UPDATE `tbl_users` SET `id`=:yp0, `username`=:yp1, `password`=:yp2, `social_id`=:yp3, `social_token`=:yp4, `user_token`=:yp5, `social_token_type`=:yp6, `email`=:yp7, `phone`=:yp8, `create_time`=:yp9, `lastaction`=:yp10, `status`=:yp11, `city`=:yp12, `country`=:yp13, `gender`=:yp14, `ip`=:yp15, `avatar`=:yp16, `whatsup`=:yp17, `favorites`=:yp18, `geolocation`=:yp19, `range`=:yp20, `current`=:yp21 WHERE `tbl_users`.`id`=187.
2016/05/26 07:02:23 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\xA4\x94 f...' for column 'whatsup' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1753): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1127): CActiveRecord->updateByPk('187', Array)
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->update(NULL)
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1522): CActiveRecord->save()
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionReturnInfo()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('returninfo')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/returnInfo')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/returnInfo?user_token=27cdb6f6493c0244a5a7a425e09d3e32&favorites=French%2CPasta%2CPizza&whatsup=%F0%9F%A4%94%20friends%3F%20&current=Security%2Fcollege%20&callback=jQuery111105774273385759443_1464246102708&_=1464246102722
---
2016/05/26 07:27:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "azenv.php".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('azenv.php')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=http://check.proxyradar.com/azenv.php?auth=146424765645&a=PSCMN&i=1747555564&p=80
HTTP_REFERER=https://proxyradar.com/
---
2016/05/26 12:03:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "azenv.php".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('azenv.php')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=http://check.proxyradar.com/azenv.php?auth=146426423615&a=PSCMN&i=1747555564&p=80
HTTP_REFERER=https://proxyradar.com/
---
2016/05/26 13:36:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "testproxy.php".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('testproxy.php')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=http://testp1.piwo.pila.pl/testproxy.php
---
2016/05/26 15:53:50 [error] [exception.Services_Twilio_RestException] exception 'Services_Twilio_RestException' with message 'The 'To' number +13718194568 is not a valid phone number.' in /var/www/html/webapp/protected/extensions/twilio/Services/Twilio.php:297
Stack trace:
#0 /var/www/html/webapp/protected/extensions/twilio/Services/Twilio.php(180): Base_Services_Twilio->_processResponse(Array)
#1 /var/www/html/webapp/protected/extensions/twilio/Services/Twilio/ListResource.php(92): Base_Services_Twilio->createData('/2010-04-01/Acc...', Array)
#2 /var/www/html/webapp/protected/extensions/twilio/Services/Twilio/Rest/Messages.php(24): Services_Twilio_ListResource->_create(Array)
#3 /var/www/html/webapp/protected/extensions/twilio/Services/Twilio/Rest/Messages.php(71): Services_Twilio_Rest_Messages->create(Array)
#4 /var/www/html/webapp/protected/controllers/SiteController.php(479): Services_Twilio_Rest_Messages->sendMessage('+13472208626', '+13718194568', 'Hi, Edward Cui ...')
#5 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionInvitefriends()
#6 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('invitefriends')
#10 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/Invitefrie...')
#11 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#12 /var/www/html/webapp/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/site/Invitefriends?user_token=06bfb71a8e797999d43397a6af299472&number=13718194568&callback=jQuery111106757552535273135_1464278020599&_=1464278020618
---
2016/05/26 15:53:50 [error] [exception.Services_Twilio_RestException] exception 'Services_Twilio_RestException' with message 'The 'To' number +84812908 is not a valid phone number.' in /var/www/html/webapp/protected/extensions/twilio/Services/Twilio.php:297
Stack trace:
#0 /var/www/html/webapp/protected/extensions/twilio/Services/Twilio.php(180): Base_Services_Twilio->_processResponse(Array)
#1 /var/www/html/webapp/protected/extensions/twilio/Services/Twilio/ListResource.php(92): Base_Services_Twilio->createData('/2010-04-01/Acc...', Array)
#2 /var/www/html/webapp/protected/extensions/twilio/Services/Twilio/Rest/Messages.php(24): Services_Twilio_ListResource->_create(Array)
#3 /var/www/html/webapp/protected/extensions/twilio/Services/Twilio/Rest/Messages.php(71): Services_Twilio_Rest_Messages->create(Array)
#4 /var/www/html/webapp/protected/controllers/SiteController.php(479): Services_Twilio_Rest_Messages->sendMessage('+13472208626', '+84812908', 'Hi, Edward Cui ...')
#5 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionInvitefriends()
#6 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#7 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#8 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('invitefriends')
#10 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/Invitefrie...')
#11 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#12 /var/www/html/webapp/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/site/Invitefriends?user_token=06bfb71a8e797999d43397a6af299472&number=84812908&callback=jQuery111106757552535273135_1464278020587&_=1464278020619
---
2016/05/26 16:23:09 [error] [php] Undefined index: email (/var/www/html/webapp/protected/controllers/SiteController.php:155)
Stack trace:
#0 /var/www/html/yii/framework/web/CController.php(286): SiteController->runAction()
#1 /var/www/html/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#2 /var/www/html/yii/framework/web/CWebApplication.php(282): SiteController->run()
#3 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#4 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#5 /var/www/html/webapp/index.php(13): CWebApplication->run()
REQUEST_URI=/site/facebookLogin?callback=jQuery1111046283421479165554_1464279766129&selfData=%7B%22accessToken%22%3A%22EAAYDK8HZAzicBAHWZAL5Y5exTKLJbaZAAC1XCHMVdOQrpAOnsuZBUX9HTW9eKd6h3YMjMsDuZAw5wI9h3e636FHibjaa3rje4ZCg8h1xAdT4Hcnc53AQZALXVNWGmuC3p7wOzOB8nLpWzsf1A1YhMf1Cs1PaKFL6VhcBT3IT48UyIgD3yEhEycY%22%2C%22userId%22%3A%221175156355863179%22%2C%22name%22%3A%22Zhongtian+Xu%22%2C%22friends%22%3A%5B%7B%22name%22%3A%22Bailey+Xiong%22%2C%22id%22%3A%2210206239681729248%22%7D%2C%7B%22name%22%3A%22Shan+Ji%22%2C%22id%22%3A%221338022172878598%22%7D%2C%7B%22name%22%3A%22Jiaming+Zhong%22%2C%22id%22%3A%221155566484477457%22%7D%2C%7B%22name%22%3A%22Daniel+Zhang%22%2C%22id%22%3A%221179411452111835%22%7D%2C%7B%22name%22%3A%22Juzhou+Chen%22%2C%22id%22%3A%22981836528589818%22%7D%5D%7D&_=1464279766130
2016/05/26 16:24:22 [error] [php] Undefined index: email (/var/www/html/webapp/protected/controllers/SiteController.php:155)
Stack trace:
#0 /var/www/html/yii/framework/web/CController.php(286): SiteController->runAction()
#1 /var/www/html/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#2 /var/www/html/yii/framework/web/CWebApplication.php(282): SiteController->run()
#3 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#4 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#5 /var/www/html/webapp/index.php(13): CWebApplication->run()
REQUEST_URI=/site/facebookLogin?callback=jQuery111107096414740663022_1464279837565&selfData=%7B%22accessToken%22%3A%22EAAYDK8HZAzicBAIv2ZCZCsSxy9VMyRBe7yUoWYxPGpe26wPzApyVq5SWtzVZAQmpc2VnBI0ogd1rZAumqMAE2lKimqklqrZB1H1nT5ZAAKFKUiXjn2iThTXEvHDBoZBQuRcWD9ExZBkERr3ekYYbKmzns9YXaqCeh3qQMd2z8IZC9glxvz9C17ueGM3Y4OLfZBiyUcakPyTCxOHAbjdKDoXYdOG%22%2C%22userId%22%3A%221175156355863179%22%2C%22name%22%3A%22Zhongtian+Xu%22%2C%22friends%22%3A%5B%7B%22name%22%3A%22Bailey+Xiong%22%2C%22id%22%3A%2210206239681729248%22%7D%2C%7B%22name%22%3A%22Shan+Ji%22%2C%22id%22%3A%221338022172878598%22%7D%2C%7B%22name%22%3A%22Jiaming+Zhong%22%2C%22id%22%3A%221155566484477457%22%7D%2C%7B%22name%22%3A%22Daniel+Zhang%22%2C%22id%22%3A%221179411452111835%22%7D%2C%7B%22name%22%3A%22Juzhou+Chen%22%2C%22id%22%3A%22981836528589818%22%7D%5D%7D&_=1464279837566
2016/05/26 16:35:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "azenv.php".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('azenv.php')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=http://check.proxyradar.com/azenv.php?auth=146428050815&a=PSCMN&i=1747555564&p=80
HTTP_REFERER=https://proxyradar.com/
---
2016/05/26 21:03:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://gettimi.com/site/downloadAndroid
---
2016/05/26 21:09:06 [error] [php] Undefined index: email (/var/www/html/webapp/protected/controllers/SiteController.php:155)
Stack trace:
#0 /var/www/html/yii/framework/web/CController.php(286): SiteController->runAction()
#1 /var/www/html/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#2 /var/www/html/yii/framework/web/CWebApplication.php(282): SiteController->run()
#3 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#4 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#5 /var/www/html/webapp/index.php(13): CWebApplication->run()
REQUEST_URI=/site/facebookLogin?callback=jQuery11110244062265381217_1464296937517&selfData=%7B%22accessToken%22%3A%22EAAYDK8HZAzicBALJGtZA0J2G2SxoUgSnp6L8VY8YvInFCznAQM8H4N6wG2Ohn19XuBO0fFD1LR1doVq46m9Xg93T21N3pgpzvkSJWbTY1dXdMZBrOsLdZBrxkIMwZATtogRtBGmJFrlhp29rjEZBjkbwIupx7hd0VcECv2CNFGYrXFeZByH0RZAwKR9etjVxizoR93Xih8e0BkT3Kgn1D7KE%22%2C%22userId%22%3A%22258117247876121%22%2C%22name%22%3A%22Ziqiu+Guo%22%2C%22friends%22%3A%5B%7B%22name%22%3A%22Hezi+Jiang%22%2C%22id%22%3A%2210207755369736209%22%7D%2C%7B%22name%22%3A%22Bailey+Xiong%22%2C%22id%22%3A%2210206239681729248%22%7D%2C%7B%22name%22%3A%22Chucheng+Feng%22%2C%22id%22%3A%2210209952382274525%22%7D%2C%7B%22name%22%3A%22Jiaming+Zhong%22%2C%22id%22%3A%221155566484477457%22%7D%2C%7B%22name%22%3A%22Liu+Cong%22%2C%22id%22%3A%221010248882362402%22%7D%2C%7B%22name%22%3A%22Catherine+Y.+Zhai%22%2C%22id%22%3A%221034055036675166%22%7D%2C%7B%22name%22%3A%22Salinda+Sa+Wang%22%2C%22id%22%3A%221023752431041690%22%7D%2C%7B%22name%22%3A%22Derek+Guoshu+Li%22%2C%22id%22%3A%22959888547443076%22%7D%2C%7B%22name%22%3A%22Lai+Wei%22%2C%22id%22%3A%22737546803014742%22%7D%2C%7B%22name%22%3A%22Kelly+Siyao+Liu%22%2C%22id%22%3A%22854690064635920%22%7D%2C%7B%22name%22%3A%22Bell+Zhong%22%2C%22id%22%3A%221713761395579233%22%7D%2C%7B%22name%22%3A%22Siyu+Tang%22%2C%22id%22%3A%22272229449790641%22%7D%5D%7D&_=1464296937518
2016/05/26 21:10:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "azenv.php".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('azenv.php')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=http://check.proxyradar.com/azenv.php?auth=146429703959&a=PSCMN&i=1747555564&p=80
HTTP_REFERER=https://proxyradar.com/
---
2016/05/26 21:41:21 [error] [php] Undefined index: email (/var/www/html/webapp/protected/controllers/SiteController.php:155)
Stack trace:
#0 /var/www/html/yii/framework/web/CController.php(286): SiteController->runAction()
#1 /var/www/html/yii/framework/web/CController.php(265): SiteController->runActionWithFilters()
#2 /var/www/html/yii/framework/web/CWebApplication.php(282): SiteController->run()
#3 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController()
#4 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#5 /var/www/html/webapp/index.php(13): CWebApplication->run()
REQUEST_URI=/site/facebookLogin?callback=jQuery111106035350195597857_1464298871185&selfData=%7B%22accessToken%22%3A%22EAAYDK8HZAzicBAAketDJZAgvoO2MnnjLWZC7Y0ZCO1TEjgbPTxdPFIDZC4DZA7kw82FmEVDPU9VtcACaCCpCecuDeXFogyTsyQJUZAJPGizRrmuZCZBciDPzxbZCymjYD3sWUI1Dj4LbVPvCjQSZBTmuRXKPHPTUw34BF2eiCrsvmqzFSeIvDNLM9xnNzdskDPHTh261dvXNeoSNQkGW96gPDbE%22%2C%22userId%22%3A%22258117247876121%22%2C%22name%22%3A%22Ziqiu+Guo%22%2C%22friends%22%3A%5B%7B%22name%22%3A%22Hezi+Jiang%22%2C%22id%22%3A%2210207755369736209%22%7D%2C%7B%22name%22%3A%22Bailey+Xiong%22%2C%22id%22%3A%2210206239681729248%22%7D%2C%7B%22name%22%3A%22Chucheng+Feng%22%2C%22id%22%3A%2210209952382274525%22%7D%2C%7B%22name%22%3A%22Jiaming+Zhong%22%2C%22id%22%3A%221155566484477457%22%7D%2C%7B%22name%22%3A%22Liu+Cong%22%2C%22id%22%3A%221010248882362402%22%7D%2C%7B%22name%22%3A%22Catherine+Y.+Zhai%22%2C%22id%22%3A%221034055036675166%22%7D%2C%7B%22name%22%3A%22Salinda+Sa+Wang%22%2C%22id%22%3A%221023752431041690%22%7D%2C%7B%22name%22%3A%22Derek+Guoshu+Li%22%2C%22id%22%3A%22959888547443076%22%7D%2C%7B%22name%22%3A%22Lai+Wei%22%2C%22id%22%3A%22737546803014742%22%7D%2C%7B%22name%22%3A%22Kelly+Siyao+Liu%22%2C%22id%22%3A%22854690064635920%22%7D%2C%7B%22name%22%3A%22Bell+Zhong%22%2C%22id%22%3A%221713761395579233%22%7D%2C%7B%22name%22%3A%22Siyu+Tang%22%2C%22id%22%3A%22272229449790641%22%7D%5D%7D&_=1464298871186
2016/05/26 23:02:02 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\x8D\x91' for column 'name' at row 1. The SQL statement executed was: INSERT INTO `tbl_phone_contacts` (`name`, `email`, `number1`, `number2`, `number3`, `invited`, `signed_up`, `user_id`, `create_time`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8).
2016/05/26 23:02:02 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\x8D\x91' for column 'name' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1080): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 /var/www/html/webapp/protected/controllers/SiteController.php(427): CActiveRecord->save()
#3 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionTakePhoneContact()
#4 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('takephonecontac...')
#8 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/TakePhoneC...')
#9 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#10 /var/www/html/webapp/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/site/TakePhoneContact
---
2016/05/27 01:42:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "azenv.php".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('azenv.php')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=http://check.proxyradar.com/azenv.php?auth=146431333921&a=PSCMN&i=1747555564&p=80
HTTP_REFERER=https://proxyradar.com/
---
2016/05/27 01:43:50 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read = 0 LIMIT 1' at line 1. The SQL statement executed was: SELECT * FROM `tbl_requests` `t` WHERE trash = 0 AND request_day = :request_day AND request_time = :request_time AND (receiver = :me OR sender = :me) AND status = 1 AND read = 0 LIMIT 1.
2016/05/27 01:43:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read = 0 LIMIT 1' at line 1' in /var/www/html/yii/framework/db/CDbCommand.php:543
Stack trace:
#0 /var/www/html/yii/framework/db/CDbCommand.php(415): CDbCommand->queryInternal('fetch', Array, Array)
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1356): CDbCommand->queryRow()
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1461): CActiveRecord->query(Object(CDbCriteria))
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1100): CActiveRecord->find('trash = 0 AND r...', Array)
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionGetMySchedule()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('getmyschedule')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/GetMySched...')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/GetMySchedule?user_token=fc14487e19bc119e5e1115994bf094df&day=4&callback=jQuery1111047351957902659425_1464296359327&_=1464296359333
---
2016/05/27 01:44:14 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read = 0 LIMIT 1' at line 1. The SQL statement executed was: SELECT * FROM `tbl_requests` `t` WHERE trash = 0 AND request_day = :request_day AND request_time = :request_time AND (receiver = :me OR sender = :me) AND status = 1 AND read = 0 LIMIT 1.
2016/05/27 01:44:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read = 0 LIMIT 1' at line 1' in /var/www/html/yii/framework/db/CDbCommand.php:543
Stack trace:
#0 /var/www/html/yii/framework/db/CDbCommand.php(415): CDbCommand->queryInternal('fetch', Array, Array)
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1356): CDbCommand->queryRow()
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1461): CActiveRecord->query(Object(CDbCriteria))
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1100): CActiveRecord->find('trash = 0 AND r...', Array)
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionGetMySchedule()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('getmyschedule')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/GetMySched...')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/GetMySchedule?user_token=cf40d87a845c66efbbe3a73205de5029&day=4&callback=jQuery1111047452973341569304_1464312600194&_=1464312600235
---
2016/05/27 01:44:14 [error] [system.db.CDbCommand] CDbCommand::fetch() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read = 0 LIMIT 1' at line 1. The SQL statement executed was: SELECT * FROM `tbl_requests` `t` WHERE trash = 0 AND request_day = :request_day AND request_time = :request_time AND (receiver = :me OR sender = :me) AND status = 1 AND read = 0 LIMIT 1.
2016/05/27 01:44:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'read = 0 LIMIT 1' at line 1' in /var/www/html/yii/framework/db/CDbCommand.php:543
Stack trace:
#0 /var/www/html/yii/framework/db/CDbCommand.php(415): CDbCommand->queryInternal('fetch', Array, Array)
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1356): CDbCommand->queryRow()
#2 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1461): CActiveRecord->query(Object(CDbCriteria))
#3 /var/www/html/webapp/protected/controllers/SiteController.php(1100): CActiveRecord->find('trash = 0 AND r...', Array)
#4 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionGetMySchedule()
#5 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('getmyschedule')
#9 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/GetMySched...')
#10 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#11 /var/www/html/webapp/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/site/GetMySchedule?user_token=cf40d87a845c66efbbe3a73205de5029&day=4&callback=jQuery1111047452973341569304_1464312600198&_=1464312600237
---
2016/05/27 02:09:07 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\x98\x98\xE2\x9D...' for column 'name' at row 1. The SQL statement executed was: INSERT INTO `tbl_phone_contacts` (`name`, `email`, `number1`, `number2`, `number3`, `invited`, `signed_up`, `user_id`, `create_time`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8).
2016/05/27 02:09:07 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\x98\x98\xE2\x9D...' for column 'name' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1080): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 /var/www/html/webapp/protected/controllers/SiteController.php(427): CActiveRecord->save()
#3 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionTakePhoneContact()
#4 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('takephonecontac...')
#8 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/TakePhoneC...')
#9 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#10 /var/www/html/webapp/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/site/TakePhoneContact
---
2016/05/27 02:13:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in /var/www/html/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/html/yii/framework/web/CController.php(270): CController->missingAction('site')
#1 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('site')
#2 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/site/GetUn...')
#3 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/webapp/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/site/site/GetUnreadMatchList?user_token=fc14487e19bc119e5e1115994bf094df&callback=jQuery111108324907551936684_1464315187030&_=1464315187036
---
2016/05/27 02:13:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "site".' in /var/www/html/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/html/yii/framework/web/CController.php(270): CController->missingAction('site')
#1 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('site')
#2 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/site/GetUn...')
#3 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/webapp/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/site/site/GetUnreadMatchList?user_token=fc14487e19bc119e5e1115994bf094df&callback=jQuery111108324907551936684_1464315187030&_=1464315187036
---
2016/05/27 02:23:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "markallrequest".' in /var/www/html/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/html/yii/framework/web/CController.php(270): CController->missingAction('markallrequest')
#1 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('markallrequest')
#2 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/markAllReq...')
#3 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/webapp/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/site/markAllRequest?user_token=fc14487e19bc119e5e1115994bf094df&callback=jQuery111103693692326024278_1464315726733&_=1464315726735
---
2016/05/27 02:23:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "markallrequest".' in /var/www/html/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/html/yii/framework/web/CController.php(270): CController->missingAction('markallrequest')
#1 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('markallrequest')
#2 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/markAllReq...')
#3 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/webapp/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/site/markAllRequest?user_token=fc14487e19bc119e5e1115994bf094df&callback=jQuery111103693692326024278_1464315726733&_=1464315726735
---
2016/05/27 02:23:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "markallrequest".' in /var/www/html/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/html/yii/framework/web/CController.php(270): CController->missingAction('markallrequest')
#1 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('markallrequest')
#2 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/markAllReq...')
#3 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#4 /var/www/html/webapp/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/site/markAllRequest?user_token=fc14487e19bc119e5e1115994bf094df&callback=jQuery111103693692326024278_1464315726729&_=1464315726736
---
2016/05/27 03:11:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "admin/i18n/readme.txt".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('admin/i18n/read...')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/admin/i18n/readme.txt
---
2016/05/27 05:05:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "testproxy.php".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('testproxy.php')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=http://testp1.piwo.pila.pl/testproxy.php
---
2016/05/27 06:15:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "azenv.php".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('azenv.php')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=http://check.proxyradar.com/azenv.php?auth=146432972653&a=PSCMN&i=1747555564&p=80
HTTP_REFERER=https://proxyradar.com/
---
2016/05/27 08:56:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('contact')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/contact
---
2016/05/27 08:56:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact.html".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('contact.html')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/contact.html
---
2016/05/27 08:56:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contactus".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('contactus')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/contactus
---
2016/05/27 08:56:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact-us".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('contact-us')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/contact-us
---
2016/05/27 08:56:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact_us".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('contact_us')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/contact_us
---
2016/05/27 08:56:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about
---
2016/05/27 08:56:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about.html".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about.html')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about.html
---
2016/05/27 08:56:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "aboutus".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('aboutus')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/aboutus
---
2016/05/27 08:56:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2016/05/27 08:56:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about_us".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about_us')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about_us
---
2016/05/27 08:56:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "company".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('company')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/company
---
2016/05/27 08:56:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "support".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('support')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/support
---
2016/05/27 08:56:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "help".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('help')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/help
---
2016/05/27 08:56:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('contact')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/contact
---
2016/05/27 08:56:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact.html".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('contact.html')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/contact.html
---
2016/05/27 08:56:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contactus".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('contactus')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/contactus
---
2016/05/27 08:56:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact-us".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('contact-us')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/contact-us
---
2016/05/27 08:56:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "contact_us".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('contact_us')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/contact_us
---
2016/05/27 08:56:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about
---
2016/05/27 08:56:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about.html".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about.html')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about.html
---
2016/05/27 08:56:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "aboutus".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('aboutus')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/aboutus
---
2016/05/27 08:56:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about-us".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about-us')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about-us
---
2016/05/27 08:56:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "about_us".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('about_us')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/about_us
---
2016/05/27 08:56:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "company".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('company')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/company
---
2016/05/27 08:56:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "support".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('support')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/support
---
2016/05/27 08:56:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "help".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('help')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/help
---
2016/05/27 10:46:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "azenv.php".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('azenv.php')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=http://check.proxyradar.com/azenv.php?auth=146434601409&a=PSCMN&i=1747555564&p=80
HTTP_REFERER=https://proxyradar.com/
---
2016/05/27 16:18:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "azenv.php".' in /var/www/html/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('azenv.php')
#1 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /var/www/html/webapp/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=http://check.proxyradar.com/azenv.php?auth=146436590659&a=PSCMN&i=1747555564&p=80
HTTP_REFERER=https://proxyradar.com/
---
2016/05/27 16:37:02 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\x91\x8C' for column 'name' at row 1. The SQL statement executed was: INSERT INTO `tbl_phone_contacts` (`name`, `email`, `number1`, `number2`, `number3`, `invited`, `signed_up`, `user_id`, `create_time`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8).
2016/05/27 16:37:02 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\x91\x8C' for column 'name' at row 1' in /var/www/html/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /var/www/html/yii/framework/db/ar/CActiveRecord.php(1080): CDbCommand->execute()
#1 /var/www/html/yii/framework/db/ar/CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 /var/www/html/webapp/protected/controllers/SiteController.php(427): CActiveRecord->save()
#3 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): SiteController->actionTakePhoneContact()
#4 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('takephonecontac...')
#8 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('site/TakePhoneC...')
#9 /var/www/html/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#10 /var/www/html/webapp/index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/site/TakePhoneContact
---
